notepad/app/build.gradle

```diff
@@ -21,12 +21,13 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:26.0.2'
+    api 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
I believe this should also be `implementation`. `api` is only useful for modules that themselves expose an API (apps don't do this).